content/blogposts/2023/announcing-the-sourcegraph-app.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Announcing the Sourcegraph app"
+publishDate: 2023-03-23T10:00-07:00
+description: The Sourcegraph app, now in beta, brings code intelligence to your local machine in a lightweight, single-binary package.
```

Comment:
" . . . in a free, lightweight package"? IMO "free" is pretty exciting, and "lightweight" and "single-binary" are kinda getting at the same thing.

Reply:
I'm good with that-
```suggestion
description: The Sourcegraph app, now in beta, brings code intelligence to your local machine in a free, lightweight package.
```